### DISCUSSION TITLE
Result ocr of model PPOCRV4 for English without space

### DISCUSSION THREAD

**Discussion by BlueTeamQQ1** [PaddlePaddle/PaddleOCR#12983] [Q&A]
I have tried predictions on my local and also tried on the online demo, but the results still do not have spaces between the words. I wonder if there is a way to handle this case?
result:
![de6d191f874e47ab9d4e6884079155a7](https://github.com/PaddlePaddle/PaddleOCR/assets/63582498/4a1d96b6-0871-472a-bd72-ae6f5c46ad4a)
origin_image:
![test](https://github.com/PaddlePaddle/PaddleOCR/assets/63582498/17b369ac-dfd9-47d5-a01c-0b066ce3479b)

**Comment by asif-ca** [comment]
There is a white space issue in paddleOCR from the beginning. You can fine-tune the recognizer model on images that have white space. This will improve its performance in recognizing white spaces.

_**Images like below:**_

![000109206](https://github.com/PaddlePaddle/PaddleOCR/assets/86609190/d87a83db-ba73-4898-89db-066e24fd33d5)
![000109232](https://github.com/PaddlePaddle/PaddleOCR/assets/86609190/8d807f2a-a2f4-44f8-8b17-779d1b0f49bc)
